packages/snaps-cli/src/commands/manifest/implementation.test.ts

```diff
@@ -172,6 +179,7 @@ describe('manifest', () => {
             "chains": ["eip155:1", "eip155:2", "eip155:3"]
           }
         },
+        "platformVersion": "1.0.0",
```

Comment:
Where does this value come from in the test? Is it mocked somewhere?

Reply:
This is the default version returned by `getSnapManifest`.